core/templates/dev/head/app.js

```diff
@@ -355,6 +355,62 @@ oppia.factory('windowDimensionsService', ['$window', function($window) {
   };
 }]);
 
+// Service for sending events to Google Analytics.
+//
+// Note that events are only sent if the CAN_SEND_ANALYTICS_EVENTS flag is
+// turned on. This flag must be turned on explicitly by the application
+// owner.
+oppia.factory('siteAnalyticsService', ['$window', function($window) {
```

Comment:
Disagree. I want this service to be a seam for site analytics in general, and users of it should not need to know anything about the infrastructure beyond the seam. This means that if we ever change our analytics service provider, then we can modify this while still keeping the same service API, rather than having to update all the files where a call to the service is made.